visual-testing-app/src/application.tsx

```diff
@@ -1,72 +1,55 @@
-import { ComponentType, useEffect } from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+/// <reference path="../../node_modules/vite/types/importMeta.d.ts" />
```

Comment:
FYI: we can also use

```js
/// <reference types="vite/client" />
```

But it contains additional declarations like for `*.svg` that conflicts with what we also use in the repo. So I opted to only import the type declarations for `import.meta`, but we need to use a relative import path for that.